crates/katana/core/src/backend/mod.rs

```diff
@@ -347,6 +348,31 @@ impl Backend {
         }
     }
 
+    pub async fn get_storages_at(
+        &self,
+        storage_addresses: Vec<(FieldElement, Vec<FieldElement>)>,
+    ) -> Result<Vec<Vec<StarkFelt>>, SequencerError> {
```

Comment:
```suggestion
    ) -> Result<Vec<Vec<FieldElement>>, SequencerError> {
```

Prefer to use `FieldElement` whenever possible. Want to limit `StarkFelt` only for transaction execution, because don't want to have multiple `felt` types lying around as it may cause confusion.

Reply:
`getStorageAt` seems to be using `StarkFelt` for returned values which has hex serialize.

Reply:
### `starknet_getStorageAt`

![image](https://github.com/dojoengine/dojo/assets/11048263/8a99835c-3881-419d-ad45-905de10299a5)

### With `StarkFelt`

![image](https://github.com/dojoengine/dojo/assets/11048263/776099c3-8fa5-4822-9bd6-b35dbb57deef)

### With `FieldElement`

![image](https://github.com/dojoengine/dojo/assets/11048263/23d5c3d6-a156-430d-b62d-eb71f4225343)

Numeric `FieldElement` does diverge from `starknet_getStorageAt`, but is probably more flexible.

Let me know what you think!

Reply:
Hmmm i guess we can keep it that way for now for consistency sake, but I'm planning to refactor all the `StarkFelt`s into `FieldElement`s wherever possible.

actually we already defined an RPC compatible `felt` here https://github.com/dojoengine/dojo/blob/612350136b43fcff4bdc8275324899457d3818f4/crates/katana/rpc/src/api/starknet.rs#L18 which, yes, another `felt` type